modules/installing-gitops-operator-using-cli.adoc

```diff
@@ -31,29 +71,51 @@ spec:
 <2> Specify the name of the Operator to subscribe to.
 <3> Specify the name of the CatalogSource that provides the Operator.
 <4> The namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.
-+
+
 . Apply the `Subscription` to the cluster:
 +
 [source,terminal]
 ----
 $ oc apply -f openshift-gitops-sub.yaml
 ----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/openshift-gitops-operator created
+----
+
 . After the installation is complete, ensure that all the pods in the `openshift-gitops` namespace are running:
 +
 [source,terminal]
 ----
 $ oc get pods -n openshift-gitops
 ----
++
 .Example output
+[source,terminal]
+----
+NAME                                                      	  READY   STATUS    RESTARTS   AGE
+cluster-b5798d6f9-zr576                                   	  1/1 	  Running   0          65m
+kam-69866d7c48-8nsjv                                      	  1/1 	  Running   0          65m
+openshift-gitops-application-controller-0                 	  1/1 	  Running   0          53m
+openshift-gitops-applicationset-controller-6447b8dfdd-5ckgh       1/1 	  Running   0          65m
+openshift-gitops-dex-server-569b498bd9-vf6mr                      1/1     Running   0          65m
+openshift-gitops-redis-74bd8d7d96-49bjf                   	  1/1 	  Running   0          65m
+openshift-gitops-repo-server-c999f75d5-l4rsg              	  1/1 	  Running   0          65m
+openshift-gitops-server-5785f7668b-wj57t                  	  1/1 	  Running   0          53m
+----
+
+. Ensure that the pods in the `openshift-gitops-operator` namespace are running:
 +
 [source,terminal]
 ----
-NAME                                                      	READY   STATUS	RESTARTS   AGE
-cluster-b5798d6f9-zr576                                   	1/1 	Running   0      	65m
-kam-69866d7c48-8nsjv                                      	1/1 	Running   0      	65m
-openshift-gitops-application-controller-0                 	1/1 	Running   0      	53m
-openshift-gitops-applicationset-controller-6447b8dfdd-5ckgh 1/1 	Running   0      	65m
-openshift-gitops-redis-74bd8d7d96-49bjf                   	1/1 	Running   0      	65m
-openshift-gitops-repo-server-c999f75d5-l4rsg              	1/1 	Running   0      	65m
-openshift-gitops-server-5785f7668b-wj57t                  	1/1 	Running   0      	53m
-----
\ No newline at end of file
+$ oc get pods -n openshift-gitops-operator
```

Comment:
@Srivaralakshmi as we also have a second namespace to check now, I added it as a step as well.

Reply:
@jaideepr97 is it beneficial to the user to verify with `oc get pods -n openshift-gitops-operator` that those pods in the `openshift-gitops-operator` namespace are running, or is checking pods in `openshift-gitops` namespace enough?

Reply:
Thanks for adding that @ekristova

Reply:
@ekristova its always a good idea to verify that the operator pod is running without any issues after a new installation, so I would say its good to have this here

Reply:
Thanks @jaideepr97